pytest-selenium/regions/toc.py

```diff
@@ -0,0 +1,34 @@
+from selenium.webdriver.common.by import By
+from selenium.webdriver.common.keys import Keys
+
+from pages import content
+from pages.base import Page
+from regions.base import Region
+
+from utils.utility import WaitForTitleChange
+
+
+class TableOfContents(Region):
+    _root_locator = (By.CSS_SELECTOR, "ol")
+    _section_link_locator = (By.CSS_SELECTOR, "ol li a")
```

Comment:
is this the branch you wanted those extra selectors for? i can add them

Reply:
@TomWoodward Not really. But you can add here also.
I need data-testid `chapter` in the corresponding <li> tags.
and `unit` in the corresponding <li> tags